src/config/model.rs

```diff
@@ -26,6 +27,7 @@ pub struct Account {
     // Override
     pub name: Option<String>,
     pub downloads_dir: Option<PathBuf>,
+    pub signature_delimiter: Option<String>,
```

Comment:
What's you intention? Do you want to be able to have multiple signatures for one account?

Reply:
Have a look at https://github.com/soywod/himalaya/pull/114. This option allows users to customize the signature delimiter. The default one is `--\n ` which is a (standard)[https://en.wikipedia.org/wiki/Signature_block#Standard_delimiter].

Reply:
Ah! Now I've got it. Hm... I'd like to add a little comment for that if we merge it (and if you agree as well of course). ;)

Reply:
I added a comment on the `Config` struct.